tech/languages/csharp/csharp-ide.md

```diff
@@ -6,45 +6,36 @@ order: 2
 
 # C# IDEs in Fedora
 
-## JetBrains Rider
+| IDE | License | .NET Core | Mono | Debugger | Packaged in Fedora | Alternative download |
+|---|---|---|---|---|---|---|
+| [Eclipse](/tools/eclipse/about.html) with [aCute plugin](https://marketplace.eclipse.org/content/acute-c-edition-eclipse-ide-experimental) | [EPL](http://www.eclipse.org/legal/epl-2.0/) | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: |  |
+| [Monodevelop](http://www.monodevelop.com/) | [LGPLv2](http://www.gnu.org/licenses/lgpl-2.1.html) |  | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [flatpak](https://download.mono-project.com/repo/monodevelop.flatpakref) |
+| [JetBrains Rider](http://jetbrains.com/rider) | [Proprietary](https://www.jetbrains.com/store/license.html), free for [Education and OpenSource](https://www.jetbrains.com/store/#edition=discounts) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  | tarball |
+| [Visual Studio Code](https://code.visualstudio.com) with [C# plugin](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) | Binary is [Proprietary](https://code.visualstudio.com/License/) Source Code is [MIT](https://github.com/Microsoft/vscode/blob/master/LICENSE.txt) | :heavy_check_mark: |  | :heavy_check_mark: |  | [flatpak](https://flathub.org/repo/appstream/com.visualstudio.code.flatpakref), copr & rpm |
```

Comment:
There is really no other alternative to downloading `.flatpakref` directly?

Reply:
Additionally, you should probably point to some install guide for flatpacks (https://github.com/developer-portal/content/issues/245). Or does it work out of the box?

Reply:
On Fedora gnome - click a link - opens up in gnome software... you're done.

Reply:
I would not reference to flatpak on this page, if we do that we would have to explain what is dnf, tar, rpm... it's on the same level. Thoughts @AxelSilverdew ?

Reply:
Simply linking to https://fedoramagazine.org/getting-started-flatpak/ in some `install with Flatpack` sentence would be sufficient imho. And when you're pointing to a file, It should IMHO say `download flatpack file` or similar.
Not all developers are experienced :)

Reply:
Well, seeing as how flatpak is newer, maybe a link to the flatpak fedoramagazine article would be a good idea so that people who are **not** on gnome have something for reference.

Installing flatpaks on Fedora Gnome is seamless though. I'd say it's akin to Suse's one click install thing on gnome.

Reply:
I don't think that people who are not flatpak fans would install a flatpak IDE if they have other options...

Reply:
Well based on conversations I've had with other people, most don't even know that flatpaks are a thing. So, putting the link somewhere might raise awareness that flatpaks are a thing.